Apps/Sandcastle/gallery/Scene Rendering Performance.html

```diff
@@ -115,26 +220,33 @@
     scene.maximumRenderTimeChange = value;
 });
 
-Sandcastle.addToolbarButton('Request a new render frame', function() {
-    scene.requestRender();
-});
-
 scene.postRender.addEventListener(function() {
     var time = Cesium.JulianDate.toGregorianDate(scene.lastRenderTime);
     var value = time.hour + ':' + time.minute + ':' + time.second + ':' + time.millisecond.toFixed(0);
     Cesium.knockout.getObservable(viewModel, 'lastRenderTime')(value);
 });
 
-Sandcastle.addToolbarButton('Load a tileset', function() {
-    scene.primitives.add(new Cesium.Cesium3DTileset({
-        url : 'https://beta.cesium.com/api/assets/1459?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIzNjUyM2I5Yy01YmRhLTQ0MjktOGI0Zi02MDdmYzBjMmY0MjYiLCJpZCI6NDQsImFzc2V0cyI6WzE0NTldLCJpYXQiOjE0OTkyNjQ3ODF9.SW_rwY-ic0TwQBeiweXNqFyywoxnnUBtcVjeCmDGef4'
-    })).readyPromise.then(function(tileset) {
-        var boundingSphere = tileset.boundingSphere;
-        viewer.camera.viewBoundingSphere(boundingSphere, new Cesium.HeadingPitchRange(0.5, -0.2, boundingSphere.radius * 4.0));
-        viewer.camera.lookAtTransform(Cesium.Matrix4.IDENTITY);
-    }).otherwise(function(error) {
-        throw(error);
-    });
+var scenarios = [{
+    text : 'Default view',
+    onselect : resetScene
+}, {
+    text : 'Load a 3D tileset and set the view',
+    onselect : loadTilesetScenario
+}, {
+    text : 'Mouseover picking',
+    onselect : pickingScenario
+}, {
+    text : 'Load CZML with model and set the view',
+    onselect : loadCzmlScenario
+}, {
+    text : 'Scene changes with API',
```

Comment:
This mode should reset the camera to be away from the globe. Otherwise when going from `Load CZML with model and set the view` to `Scene changes with API` it looks like nothing changed.

Reply:
Fixed.